pkg/datapath/linux/node.go

```diff
@@ -979,6 +1034,10 @@ func (n *linuxNodeHandler) nodeUpdate(oldNode, newNode *nodeTypes.Node, firstAdd
 		return nil
 	}
 
+	if n.nodeConfig.EnableExternalVxlan {
+		updateIPCacheExternalVxlanMapping()
+	}
```

Comment:
I'm not sure I quite understand why this is attached to node update logic. I believe that this parent function is run any time any node joins or leaves the cluster, which generates updates for the k8s `Node` resource.

I suspect that it should be sufficient to push this logic into somewhere like maybe... `syncEndpointsAndHostIPs()` under `daemon/cmd/datapath.go`, like other ipcache updates.

Reply:
I am copying `updateTunnelMapping()` and my intentions is to make sure the VTEP entry always in ipcache like node podCIDR prefix always in the tunnel map, concerned if any sync problem may cause the VTEP entry get removed from ipcache, but since you think each node join or leave triggers the code and may bring extra work, I can put the logic in `syncEndpointsAndHostIPs()`